.github/workflows/release.yml

```diff
@@ -34,14 +34,19 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Add kube-state-metrics repository
-        run: helm repo add kube-state-metrics https://kubernetes.github.io/kube-state-metrics
-
-      - name: Add Pixie repository
-        run: helm repo add pixie-operator https://pixie-operator-charts.storage.googleapis.com
-
-      - name: Add newrelic-k8s-metrics-adapter repository
-        run: helm repo add newrelic-k8s-metrics-adapter https://newrelic.github.io/newrelic-k8s-metrics-adapter
+      - name: Add helm repositories
+        run: |
+          helm repo add newrelic-helm-charts https://newrelic.github.io/helm-charts
+          helm repo add newrelic-infrastructure https://newrelic.github.io/nri-kubernetes
+          helm repo add nri-prometheus https://newrelic.github.io/nri-prometheus
+          helm repo add nri-metadata-injection https://newrelic.github.io/k8s-metadata-injection
+          helm repo add newrelic-k8s-metrics-adapter https://newrelic.github.io/newrelic-k8s-metrics-adapter
+          helm repo add kube-state-metrics https://kubernetes.github.io/kube-state-metrics
+          helm repo add nri-kube-events https://newrelic.github.io/nri-kube-events
+          # helm repo add newrelic-logging  # Yet to be moved/deprecated
```

Comment:
also this, unless you are adding it for some reason

Reply:
Solved in [#4f2332e](https://github.com/newrelic/helm-charts/pull/795/commits/4f2332e6fc57c5e9842ddb8e958681644a537814)